post_installation_configuration/aws-compute-edge-tasks-zones.adoc

```diff
@@ -1,50 +1,51 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="aws-compute-edge-tasks"]
-= AWS Local Zone tasks
+[id="aws-compute-edge-tasks-zones"]
+= AWS zone tasks
 include::_attributes/common-attributes.adoc[]
-:context: aws-compute-edge-tasks
+:context: aws-compute-edge-tasks-zones
+:zone-type: Local Zones or Wavelength Zones
 
 toc::[]
 
-After installing {product-title} on Amazon Web Services (AWS), you can further configure AWS Local Zones and an edge compute pool, so that you can expand and customize your cluster to meet your needs.
+After installing {product-title} on Amazon Web Services (AWS), you can further configure AWS {zone-type} and an edge compute pool, so that you can expand and customize your cluster to meet your needs.
 
-// Extend existing clusters to use AWS Local Zones
+// Extend existing clusters to use AWS zones
 include::modules/post-install-edge-aws-extend-cluster.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
 * For more information about AWS Local Zones, the supported instances types, and services, see link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features] in the AWS documentation.
+* For more information about AWS Local Zones, the supported instances types, and services, see link:https://aws.amazon.com/wavelength/features/[AWS Wavelength features] in the AWS documentation.
 
-
-// About the Edge Compute Pool
+// Edge compute pools and AWS zones
 include::modules/edge-machine-pools-aws-local-zones.adoc[leveloffset=+2]
 
 [id="post-install-extend-existing-to-local-zones-mtu"]
-=== Changing the Cluster Network MTU to support AWS Local Zones subnets
+=== Changing the Cluster Network MTU to support AWS zone subnets
 
 You might need to change the maximum transmission unit (MTU) value for the cluster
-network so that your cluster infrastructure can support Local Zone subnets.
+network so that your cluster infrastructure can support {zone-type} subnets.
 
 // About the cluster MTU
 include::modules/nw-cluster-mtu-change-about.adoc[leveloffset=+3]
 
-// Changing the cluster MTU
+// Changing the cluster MTU -- CONFIRM WITH MARCO as steps differ
 include::modules/nw-cluster-mtu-change.adoc[leveloffset=+3]
 
-// Opting in to AWS Local Zones
-include::modules/installation-aws-add-local-zone-locations.adoc[leveloffset=+2]
+// Opting in to an AWS zone
+include::modules/installation-aws-add-zone-locations.adoc[leveloffset=+2]
 
-// Extend existing clusters to use AWS Local Zones
-include::modules/post-install-existing-local-zone-subnet.adoc[leveloffset=+2]
+// Create network requirements in an existing VPC to use an AWS zone
+include::modules/post-install-extend-existing-to-zone-type.adoc[leveloffset=+2]
 
-// Creating a subnet in AWS Local Zones
-include::modules/installation-creating-aws-subnet-localzone.adoc[leveloffset=+3]
+// Creating a subnet in an AWS zone
+include::modules/installation-creating-aws-subnet-localzone.adoc[leveloffset=+2]
```

Comment:
It seems you need to revisit this part. The "Task book", section to create the subnet, must follow the similar steps of Wavelength Zones:
https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-wavelength-zone#installation-creating-aws-vpc-subnets_installing-aws-wavelength-zone

The rendered steps are wrong, the parameters file missmatch the cloudformation template params:

https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/aws-compute-edge-tasks-zones#installation-creating-aws-subnet-localzone_aws-compute-edge-tasks-zones

IMO the good approach here is to follow the latest content, used in WL book, which is valid for the "task book"

Reply:
Hi @mtulio . I removed the module with the incorrect information anf placed the WZ module in the AWS edge zone tasks doc book. Does the information in [link](https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-wavelength-zone#installation-creating-aws-vpc-subnets_installing-aws-wavelength-zone) also apply to Local Zones?